components/TextFields/README.md

```diff
@@ -142,6 +142,91 @@ import MaterialComponents.MaterialTextFields
 ```
 <!--</div>-->
 
+### Theming
+
+You can theme textFields components with your app's color scheme and typography scheme using the ColorThemer and TypographyThemer extensions.
+
+You must first add the Themer extensions to your project:
+
+``` bash
+pod 'MaterialComponents/TextFields+Extensions/ColorThemer'
+pod 'MaterialComponents/TextFields+Extensions/TypographyThemer'
+```
+
+You can then import the theming APIs:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+import MaterialComponents.MaterialTextFields_ColorThemer
+import MaterialComponents.MaterialTextFields_FontThemer
+```
+
+#### Objective-C
+
+``` objc
+#import "MaterialTextFields+ColorThemer.h"
+#import "MaterialTextFields+TypographyThemer.h"
+```
+<!--</div>-->
+
+MDCTextFieldColorThemer allows you to theme a TextInput or an InputController with your app's color scheme. 
```

Comment:
Should this be, MDCTextInputController, MDCTextField, or MDCMultilineTextField instead of TextInput or InputController?